spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/model/impl/MultiIRI.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.rdf4j.model.impl;
```

Comment:
This and other classes will need the RDF4J copyright header added. It should be this:

```
/*******************************************************************************
 * Copyright (c) 2020 Eclipse RDF4J contributors.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *******************************************************************************/
```

Reply:
ok

Reply:
done